### ISSUE TITLE
"Hide products on sales page that you are not interested in" don't work in Firefox

### ISSUE THREAD

**Issue by Yellow-Horror** [SteamDatabase/BrowserExtension#9]
Firefox version 52.2.1 (32-бит)
The games i mark "not interested" in Steam are just marked with crossed circle but not hide on Steam Sales page.

**Comment by xPaw** [comment]
Yeah I did not yet re-implement this on the website after I moved this code to the website.

**Comment by xPaw** [comment]
Fixed now.

**Comment by ourichermath** [comment]
I'm logged into both SteamDB (with my Steam credentials) and the Steam Store on my browser.  In my browser, if I look at the store pages for the items I've marked as "Not Interested," they are actually showing as "Not Interested."  Ditto for doing that within Steam, itself.  I'm allowing cookies but blocking 3rd party cookies.  But, I've got store.steampowered.com on my cookie whitelist.  Is there something else I need to set?  Is there some way to force a refresh of whatever Steam Database is using to know the "Not Interested" status?
